CHANGELOG.md

```diff
@@ -13,9 +14,11 @@ Two faster strategies have been added for this use case: `xxh64` and
 lightweight. Read more about it in the [call caching documentation](
 https://cromwell.readthedocs.io/en/stable/Configuring/#call-caching).
 
-### Bugfixes
+### Bug fixes
 
-+ Fixed a bug where the `use_relative_output_paths` option would not preserve intermediate folders.
+* Fixed a bug where the `size(...)` function did not work correctly on files from the input section
+  of a task that had a relative path.
+* Fixed a bug where the `use_relative_output_paths` option would not preserve intermediate folders.
```

Comment:
```suggestion
* Fixed a bug where the `use_relative_output_paths` option would not preserve intermediate folders in shared filesystem backends.
```

Reply:
This change got already merged. I have merged the develop branch into this branch and made sure it does not show up as a change anymore. Said change affected all backends by the way.